app/src/main/kotlin/com/hippo/ehviewer/ui/reader/BitmapPainter.kt

```diff
@@ -4,39 +4,34 @@ import android.graphics.Bitmap
 import android.graphics.Matrix
 import android.graphics.Paint
 import android.graphics.RectF
-import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Rect
 import androidx.compose.ui.graphics.drawscope.DrawScope
 import androidx.compose.ui.graphics.drawscope.drawIntoCanvas
 import androidx.compose.ui.graphics.nativeCanvas
 import androidx.compose.ui.graphics.painter.Painter
-import androidx.compose.ui.unit.IntRect
+import androidx.compose.ui.graphics.toAndroidRectF
+import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.unit.toSize
 import androidx.compose.ui.util.fastRoundToInt
 
-class BitmapPainter(private val bitmap: Bitmap, srcRect: IntRect) : Painter() {
-
-    private val srcRect = with(srcRect) {
-        RectF(left.toFloat(), top.toFloat(), right.toFloat(), bottom.toFloat())
-    }
-
+class BitmapPainter(private val bitmap: Bitmap) : Painter() {
+    override val intrinsicSize = IntSize(bitmap.width, bitmap.height).toSize()
+    private val srcRect = Rect(Offset.Zero, intrinsicSize).toAndroidRectF()
```

Comment:
Can use `Size.toRect()`, though it will create an intermediate object comparing to construct `RectF` directly.

Reply:
Done